web/src/app/app-service/app-service-configurable/insecure-secrets/insecure-secrets.component.html

```diff
@@ -17,6 +17,12 @@
  -->
 
  <div *ngIf="insecureSecrets">
+    <div class="alert alert-info mb-3" role="alert">
+      <small>
+          <i class="fa fa-info-circle"></i> 
+          Insecure Secrets that are used when running in non-secure mode, i.e. when Vault isn't available. This is a dynamic map of configuration, so can empty if no secrets are used or can have as many or few user define secrets. It simulates a Secret Store in non-secure mode. 
```

Comment:
```suggestion
          Insecure Secrets that are used when running in non-secure mode, i.e. when Vault isn't available. This is a dynamic map of configuration, so can empty if no secrets are used or can have as many or few user-defined secrets. It simulates a Secret Store in non-secure mode. 
```

Reply:
this messaga is copied from docs.edgexfoundry.org, but I can update it if you think it's not right

Reply:
Your PR. Your decision.  I didn't block the merge.